CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+
+* The `main` and `develop` branches are locked for the push action.
+* Releases are created from the `main` branch. If you have Azure Integration set up, it is running code from the `main` branch. Pull requests into the `main` branch are not accepted.
+* The `develop` branch functions as a candidate for the next release. Create your pull requests into this branch. The code in `develop` must always pass the tests.
+
+### How to build
+* After cloning the repository, run `yarn install` to install dependencies.
+
+* Run `yarn build` to build the project into the `dist` folder. The created `dist/fingerprintjs-pro-azure-function/fingerprintjs-pro-azure-function.js` and `dist/fingerprintjs-pro-azure-function-management/fingerprintjs-pro-azure-function-management.js` are meant to be deployed to Azure Functions.
+
+### How to deploy to Azure Functions
+
+1. Install the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) and sign in to Azure account according to the [Sign in with Azure CLI](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli) guide.
+
+2. Install the [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Cmacos%2Ccsharp%2Cportal%2Cbash#install-the-azure-functions-core-tools).
+
+3. Build the project.
+
+4. Go to `dist` directory and run
+    ```shell
+    func azure functionapp publish <AppFunctionName>
+    ```
+    where `<AppFunctionName> is a Azure Function App name in the Azure account.
+
+
+You can invoke the function from your local environment using the [start command](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Cmacos%2Ccsharp%2Cportal%2Cbash#start) in Azure Functions Core Tools.
+
+
+### How to run function locally
+
+* You can run and debug the function locally via [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-develop-local). The project has a command `yarn start` to do that.
+* [Azurite](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio) could be used to emulate Azure Storage account. The project's command is `yarn emulate-storage`. 
+
+
+### Code style
+
+Consistent code formatting is enforced by [ESLint](https://eslint.org/) and [Prettier](https://prettier.io/). To check your code, run:
+```shell
+yarn lint
+```
+
+You don't need to do this manually, CI runs the check automatically. To fix all auto-fixable issues at once, run:
+```shell
+yarn lint:fix
+```
+
+### Commit style
+
+You are required to follow [conventional commits](https://www.conventionalcommits.org) rules.
+
+### How to release a new version
+
+The workflow `release.yml` is responsible for releasing a new version. It has to be run on `develop` branch, and at the end it will create a release and a PR to `main` branch.
```

Comment:
```suggestion
The workflow `release.yml` is responsible for releasing a new version. It has to be run on the `develop` branch, and at the end, it will create a release and a PR to the `main` branch.
```